docs/how_to_guides/validation/how_to_add_a_validation_operator.rst

```diff
@@ -0,0 +1,68 @@
+.. _how_to_guides__validation__how_to_add_a_validation_operator:
+
+How to add a Validation Operator
+======================================
+
+This guide will help you add a new instance of a :ref:`Validation Operator <validation_operators_and_actions>`. Validation Operators give you the ability to validate multiple batches
+of data together. Additionally, each Validation Operator encodes a set of business rules around validation. When you add an instance of a Validation Operator,
+you can use its business rules in your project.
```

Comment:
Minor tweak on review wording here: I think we're trying to be normative wrt usage, so "should be rare" instead of "is rare" is the intent. That said -- I think rather than rare/not rare the key question should remain the one that motivated naming of VOs in the beginning -- side effects. If you want side effects...including storing the result...then you want a VO, whether through a checkpoint or not.

Reply:
The opening paragraph is updated. @jcampbell , if you have suggestions for a change, we can do it as a separate PR.